x-pack/plugins/security_solution/public/resolver/view/graph_controls.test.tsx

```diff
@@ -0,0 +1,255 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { Simulator } from '../test_utilities/simulator';
+import { noAncestorsTwoChildren } from '../data_access_layer/mocks/no_ancestors_two_children';
+import { nudgeAnimationDuration } from '../store/camera/scaling_constants';
+import '../test_utilities/extend_jest';
+import { ReactWrapper } from 'enzyme';
+
+describe('graph controls', () => {
+  let simulator: Simulator;
+  let originEntityID: string;
+  let originNode: ReactWrapper;
+  const resolverComponentInstanceID = 'graph-controls-test';
+
+  beforeEach(async () => {
+    const {
+      metadata: { databaseDocumentID, entityIDs },
+      dataAccessLayer,
+    } = noAncestorsTwoChildren();
+
+    simulator = new Simulator({
+      dataAccessLayer,
+      databaseDocumentID,
+      resolverComponentInstanceID,
+    });
+    originEntityID = entityIDs.origin;
+  });
+
+  describe('when the graph controls load', () => {
+    it('should display all cardinal panning buttons and the center button', () => {
+      const westPanButton = simulator
+        .graphControlElement()
+        .find('[data-test-subj="resolver:graph-controls:west-button"]');
```

Comment:
`.find` will return react components as well as DOM nodes. In order to test the DOM *only* you need to use `domNodes`: https://github.com/elastic/kibana/blob/master/x-pack/plugins/security_solution/public/resolver/test_utilities/simulator/index.tsx#L246

Would you consider using that instead?

Also, these tests would be more resilient to change if you use `.map`. As it is, these pass  because Resolver shows the controls immediately on the first render. If we (for example) made the controls show up after data had loaded then these tests would likely fail. They might even fail now if you make the mock data access layer resolve the API calls in the next event-loop cycle.

For that reason, would you consider using `.map`?

Reply:
Or consider `resolveWrapper` (uses `map` indirectly)

Reply:
Yea, I forgot about that concern when I wrote this tbh. Updated and refactored to use `.map` and `domNodes`. Thanks!